samples/iotc-basic/src/main.c

```diff
@@ -19,7 +19,7 @@
 #include "led_pwm.h"
 #else
 #define ui_leds_init()
-#define ui_led_set_rgb(a,b,c) (void) (a,b,c)
+#define ui_led_set_rgb(a,b,c)
```

Comment:
why are the void statements removed?

Reply:
https://github.com/avnet-iotconnect/iotc-nrf-sdk/commit/2e1de91e2c0518b1d6b3767504b08939b5b282e7 

updated in the above latest commit.

Reply:
#define ui_led_set_rgb(a,b,c) was left since i used it from releases https://github.com/avnet-iotconnect/iotc-nrf-sdk/releases/tag/v2.0.0  "v2.0.0".